canonical/ubuntu.com
# WD-16643 Rebrand /internet-of-things/gateways

## Done

- Rebranded /internet-of-things/gateways
- There may be some changes to content, please refer copydoc

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [Demo](https://ubuntu-com-14536.demos.haus/internet-of-things/gateways)
- [Design](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=0-1&node-type=canvas&m=dev)
- [Copydoc](https://docs.google.com/document/d/18ffTcNolrtJS9AvjMfa8KW94uLfC0KzmHtTe-jMYmlo/edit?tab=t.0)
- This is a big page, please look for incorrect links and content dicrepancies.

## Issue / Card

Fixes # [WD-16643](https://warthogs.atlassian.net/browse/WD-16643)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-16643]: https://warthogs.atlassian.net/browse/WD-16643?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
